javalib/src/main/scala/java/util/SplittableRandom.scala

```diff
@@ -0,0 +1,136 @@
+package java.util
+
+/*
+ * This is a clean room implementation derived from the original paper
+ * and Java implementation mentioned there:
+ *
+ * Fast Splittable Pseudorandom Number Generators
+ * by Guy L. Steele Jr., Doug Lea, Christine H. Flood
+ * http://gee.cs.oswego.edu/dl/papers/oopsla14.pdf
+ *
+ */
+private object SplittableRandom {
+
+  private final val DOUBLE_ULP = 1.0 / (1L << 53)
+  private final val GOLDEN_GAMMA = 0x9e3779b97f4a7c15L // odd
+
+  //switched from original comparing result of JVM std lib
+  private final def mix64variant13(_z: Long): Long = {
+  //def mix64(_z: Long): Long = {
```

Comment:
Calling this function `mix64variant13` is nonsense: it's the original MumurHash3 finalizer, not "variant 13". I suggest you call it `mix64ForGamma` instead, since it is the function used in `mixGamma`. The comment should say that it is in fact the original MurmurHash3 finalizer function, whereas the paper version uses variant 13 for this purpose.